src/test/java/dk/kb/present/transform/XSLTSchemaDotOrgTransformerTest.java

```diff
@@ -63,16 +69,116 @@ void testCreatorsAndHeadline() throws Exception {
 
+        String correctString = importTestFile("src/test/resources/schemaOrgJsonTestFiles/schemaOrg_DPK000107.json");
+        Assertions.assertEquals(schemaOrgString, correctString);
+    }
+
 
-        Assertions.assertTrue(schemaOrgString.contains("\"id\":\"770379f0-8a0d-11e1-805f-0016357f605f\""));
-        Assertions.assertTrue(schemaOrgString.contains("\"dateCreated\":\"1974\""));
-        Assertions.assertTrue(schemaOrgString.contains("\"creator\":[{\"@type\":\"Person\",\"name\":\"John R. (John Rosforth) Johnsen\",\"givenName\":\"John R. (John Rosforth)\",\"familyName\":\"Johnsen\",\"birthDate\":\"1945-0-0\",\"deathDate\":\"2016-0-0\",\"hasOccupation\":{\"@type\":\"Occupation\",\"name\":\"fotograf\"}}]"));
-        Assertions.assertTrue(schemaOrgString.contains("\"headline\":[{\"value\":\"Romeo og Julie\",\"@language\":\"da\"}]"));
+    @Test
+    void testMaterialSize() throws Exception {
+        String schemaOrgString = TestUtil.getTransformedWithAccessFieldsAdded(MODS2SCHEMAORG, RECORD_ANSK);
+
+        Gson gson = new GsonBuilder().setPrettyPrinting().create();
+        JsonElement je = JsonParser.parseString(schemaOrgString);
+        String prettyJsonString = gson.toJson(je);
+        //System.out.println(prettyJsonString);
+        //System.out.println(schemaOrgString);
+
+        String correctString = importTestFile("src/test/resources/schemaOrgJsonTestFiles/schemaOrg_ANSK_11614.json");
+        Assertions.assertEquals(schemaOrgString, correctString);
+    }
+
+    @Test
+    void testInternalNotesToKbAdmin() throws Exception {
+        String schemaOrgString = TestUtil.getTransformedWithAccessFieldsAdded(MODS2SCHEMAORG, RECORD_DNF);
+
+        Gson gson = new GsonBuilder().disableHtmlEscaping().setPrettyPrinting().create();
+        JsonElement je = JsonParser.parseString(schemaOrgString);
+        String prettyJsonString = gson.toJson(je);
+        //System.out.println(prettyJsonString);
+        //System.out.println(schemaOrgString);
+
+        String correctString = importTestFile("src/test/resources/schemaOrgJsonTestFiles/schemaOrg_DNF_1951-00352_00052.json");
+        Assertions.assertEquals(schemaOrgString, correctString);
+    }
+
+    @Test
+    void testImageUrlCreation () throws Exception {
+        String schemaOrgString = TestUtil.getTransformedWithAccessFieldsAdded(MODS2SCHEMAORG, RECORD_40221e);
+
+        Gson gson = new GsonBuilder().disableHtmlEscaping().setPrettyPrinting().create();
+        JsonElement je = JsonParser.parseString(schemaOrgString);
+        String prettyJsonString = gson.toJson(je);
+        //System.out.println(prettyJsonString);
+        //System.out.println(schemaOrgString);
+
+        String correctString = importTestFile("src/test/resources/schemaOrgJsonTestFiles/schemaOrg_40221e30-1414-11e9-8fb8-00505688346e.json");
+        Assertions.assertEquals(schemaOrgString, correctString);
+
+    }
+
+
+    /**
+     * Update test files when XSLT changes.
+     */
```

Comment:
I understand what is happening here. It is useful and at the same time a bit temptation to just regenerate the test files without inspecting them to see if the change was benign.

Let's discuss this face2face.